lib/kazoo/cluster.rb

```diff
@@ -70,6 +70,26 @@ def topics
       end
     end
 
+    def create_topic(name, partitions: nil, replication_factor: nil)
+      raise ArgumentError, "partitions must be set to a positive integer" if partitions.nil?
```

Comment:
Shouldn't this be `Integer(partitions) <= 0` instead? Or, even better, just make partitions a required argument seen as `nil` isn't acceptable?